### DISCUSSION TITLE
Ability to manage lazy load images in Elementor

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#22442] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi team

In website speed tests, we always face this error :  **Don't lazy load Largest Contentful Paint image**

### Describe the solution you'd like

So it would be great if you could allow us to remove the desired images from lazy load. Or, Elementor will automatically remove the images at the beginning of the page from lazy load.

**Comment by Tommy-Mixon** [comment]
The Elementor team needs to fix this. Support is acting like it's WordPress is adding lazy load and not Elementor. They have not responded to any of their customer's issues. This is unsatisfactory.
